startup/gui/bookmarks.py

```diff
@@ -45,3 +45,15 @@
 
 fontBookmarks = GafferUI.Bookmarks.acquire( application, category="font" )
 fontBookmarks.add( "Gaffer Fonts", os.path.expandvars( "$GAFFER_ROOT/fonts" ) )
+
+shaderBookmarks = GafferUI.Bookmarks.acquire( application, category="shader" )
+defaultShaderDirectory = os.path.expandvars( "$HOME/gaffer/shaders" )
+try :
```

Comment:
I guess you could also do this the opposite way round

```
if not os.path.isdir( defaultShaderDirectory ) :
      os.makedirs( defaultShaderDirectory )
```

Might not have any speed advantage, but its less verbose and doesn't require an explaining comment... anyways, I'm not too fused either way.